packages/inventory/src/Stocktake/DetailView/reducer.ts

```diff
@@ -103,6 +114,13 @@ export const reducer = (
           state.draft.onHold = !state.draft.onHold;
           break;
         }
+
+        case StocktakeActionType.UpdateStatus: {
+          // TODO: Probably there should be some more checks here for
+          // finalising
```

Comment:
as far as I can tell, mSupply only asks. 
If there aren't adjustments to make, it just finalises all the same.